examples/annotation_import/text.ipynb

```diff
@@ -76,12 +76,27 @@
     },
```

Comment:
<p>same nit as before, we should either keep as None or i'd like to know motivation for False &gt; None</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/523/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>